app/assets/stylesheets/_variables.scss

```diff
@@ -35,6 +35,6 @@ $picture-small: 75px !default;
 $picture-medium: 100px !default;
 
 //## Variables for icons or logos used in Coursemology.
-$course-achievement-badge-thumb:   $picture-thumb !default;
-$course-achievement-badge-small:   $picture-small !default;
-$course-achievement-badge-medium:  $picture-medium !default;
+$course-achievement-badge-form:   $picture-small !default;
+
+$course-logo-form: $picture-small !default;
```

Comment:
no, prefer clarity over ambiguity. of course don't make the identifier 100 chars long...